collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -88,7 +70,7 @@
     UserAgentProvider userAgentProvider;
```

Comment:
It is unused now.

Reply:
Ah nice catch. I really dislike that Dagger's generated code sometimes fools Android Studio into thinking a field is used.